internal/main.go

```diff
@@ -55,6 +57,14 @@ func main() {
 		return
 	}
 
+	if flags.validate != "" {
+		if err := exec.Validate(flags.validate); err != nil {
+			fmt.Fprintf(os.Stderr, "%v\n", err)
+			os.Exit(4)
```

Comment:
Why `4`?

Reply:
1 is for failing to run the stage, 2 is for bad command line arguments. I've seen a lot of things use powers of 2 for exit status (presumably so each error is its own bit), so I picked the next one. I'd be OK with 3 though, since we don't report simulateous errors anyway.

Reply:
I think 1 is more appropriate (general execution error). Since the user has explicitly invoked the validation functionality, they know that any error means that validation failed.